lib/mongo_mapper/plugins/stats.rb

```diff
@@ -5,8 +5,18 @@ module Plugins
     module Stats
       extend ActiveSupport::Concern
       module ClassMethods
-        def stats
-          stats = database.command(:collstats => collection.name).documents[0]
+        ALLOWED_OPTIONS = [:scale].freeze
+
+        def stats(opts={})
```

Comment:
How about an optional keyword argument like `stats(scale: nil)`?
It will be clear, and we don't need a Hash object for this method because we won't relay the options between layered methods like 'save' or `first`.
Please let me know what you think 😃

Reply:
Ultimately it's up to the maintainers, but in this case I believe that the combination of ALLOWED_OPTIONS and opts makes the stats() method unnecessarily complicated, whereas it doesn't need extensibility yet.
Maybe when mongomapper supports the 3rd kwargs, it's the time to refactor to introduce the constant. In this case the method definition is going to be like `def stats(**opts)` though

Reply:
> Maybe when mongomapper supports the 3rd kwargs, it's the time to refactor to introduce the constant.
Basically, I agree.
I was just wondering whether we should minimize the possibility for future breaking change in advance.
But I changed my mind and using keyword argument sounds better for this situation to me.

Reply:
I fixed it :)
(feel free to squash the commits when merging it)

https://github.com/mongomapper/mongomapper/pull/722/commits/a908eb88133066a9d35550f88d7ee7fc7976f2e7